packages/flutter/test/painting/continous_rectangle_border_test.dart

```diff
@@ -82,7 +82,7 @@ void main() {
         shape: const ContinuousRectangleBorder(
           borderRadius: BorderRadius.only(
             topLeft: Radius.circular(28.0),
-            bottomRight: Radius.circular(14.0),
+            bottomLeft: Radius.circular(14.0),
```

Comment:
For future reference: This is what the golden file looks like: https://flutter-gold.skia.org/img/images/9404cb8a76e0c694926a2b6cdb08181a.png

Reply:
How do you generate images like this? I want add different radius on all four corners so we can verify them all in one go.

Reply:
Hi @a9udn9u, do you mean you want to add more image tests as part of this change? I can help with that. Visit our wiki [Writing a golden file test for package:flutter](https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter) and let me know if that sounds like what you are looking for. :)